ports/curl/patches/0002-Dont-set-_USRDLL-on-a-static-Windows-build.patch

```diff
@@ -0,0 +1,32 @@
+From c70bb0ecfcf823320301dd52b1e69b5451a11915 Mon Sep 17 00:00:00 2001
+From: Don <don.j.olmstead@gmail.com>
+Date: Wed, 17 Nov 2021 13:21:28 -0800
+Subject: [PATCH] Don't set _USRDLL on a static Windows build
+
+---
+ lib/CMakeLists.txt | 5 +----
+ 1 file changed, 1 insertion(+), 4 deletions(-)
+
+diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
+index 2575288f7091..46973bf2b875 100644
+--- a/lib/CMakeLists.txt
++++ b/lib/CMakeLists.txt
+@@ -95,10 +95,6 @@ endif()
+ 
+ target_link_libraries(${LIB_NAME} ${CURL_LIBS})
+ 
+-if(WIN32)
```

Comment:
This seems to be different than what I see in https://github.com/microsoft/vcpkg/blob/master/ports/curl/0011_fix_static_build.patch assuming that's intended to be the same thing.

Is there a good reason to do it differently? It seems like it might make following future changes harder if we go our own way.

Reply:
I landed the fix in https://github.com/curl/curl/pull/8030 so it will end up there in the next release